### PR TITLE
Added some clarifying comments to ifcfg template

### PR DESCRIPTION
**Pull request by rhafer**
Also some smaller adjustments to the formatting. Hopefully slighly better
readable now.

### CODE REVIEW

File: chef/cookbooks/network/templates/default/suse-cfg.erb [hunk 1]
```diff
@@ -18,40 +18,58 @@ iface=@interfaces[@nic.name]
...
 BOOTPROTO=static
 <% end -%>
-<% unless @nic.kind_of?(Nic::Vlan) or @nic.kind_of?(Nic::Bond) or
+<%
+   # ETHTOOL_OPTIONS are only needed for "real" interfaces (i.e.
+   # everything not a bond, bridge or vlan)
+   unless @nic.kind_of?(Nic::Vlan) or @nic.kind_of?(Nic::Bond) or
        @nic.kind_of?(Nic::OvsBridge) -%>
-  <% unless @ethtool_options.empty? -%>
+<%   unless @ethtool_options.empty? -%>
```
Comment (aplanas) [inline chef/cookbooks/network/templates/default/suse-cfg.erb]: I am super glad that we add more comments, but I feel this indentation a bit weird (adding spaces inside the <% section)

Comment (rhafer) [inline chef/cookbooks/network/templates/default/suse-cfg.erb]: AFAIK that's what we do in other places as well. And at least it's now consistent throughout the whole file, which it wasn't before.

**Comment by dirkmueller** [comment]
+1

File: chef/cookbooks/network/templates/default/suse-cfg.erb [hunk 2]
```diff
@@ -18,40 +18,58 @@ iface=@interfaces[@nic.name]
...
+     when @nic.kind_of?(Nic::OvsBridge) -%>
 OVS_BRIDGE=yes
-<%   iface["slaves"].each_with_index do |slave, i| -%>
+<%     iface["slaves"].each_with_index do |slave, i| -%>
 OVS_BRIDGE_PORT_DEVICE_<%=i%>=<%=quote(slave)%>
-<%   end -%>
-<% when @nic.kind_of?(Nic::Vlan) -%>
+<%     end -%>
+<%
+   # Setting specific to vlan devices (ifcfg-vlan(5))
```
Comment (dirkmueller) [inline chef/cookbooks/network/templates/default/suse-cfg.erb]: Indentation error: should be indented two more spaces

File: chef/cookbooks/network/templates/default/suse-cfg.erb (hunk 1 above)
Comment (dirkmueller) [inline chef/cookbooks/network/templates/default/suse-cfg.erb]: not entirely consistent :-)

File: chef/cookbooks/network/templates/default/suse-cfg.erb [hunk 3]
```diff
@@ -18,40 +18,58 @@ iface=@interfaces[@nic.name]
...
-  <% unless @ethtool_options.empty? -%>
+<%   unless @ethtool_options.empty? -%>
 ETHTOOL_OPTIONS="-K iface <%= @ethtool_options -%>"
-  <% end -%>
+<%   end -%>
 <% end -%>
 <% case
-   when @nic.kind_of?(Nic::Bridge) -%>
+     # Setting specific to linuxbridge devices (ifcfg-bridge(5))
+     when @nic.kind_of?(Nic::Bridge) -%>
```
Comment (vuntz) [inline chef/cookbooks/network/templates/default/suse-cfg.erb]: Actually, `when` should be at the same level as `case` (it's confusing, but that's the style guideline).

**Comment by vuntz** [comment]
+1.

If you want to fix indentation again, you're welcome, but I wouldn't mind if it's merged without that.

File: chef/cookbooks/network/templates/default/suse-cfg.erb (hunk 1 above)
Comment (rhafer) [inline chef/cookbooks/network/templates/default/suse-cfg.erb]: I hope it's now :)

**Comment by rhafer** [comment]
> If you want to fix indentation again, you're welcome, but I wouldn't mind if it's merged without that.

I updated it again.

**Comment by aplanas** [comment]
+1

**Comment by vuntz** [comment]
+1

**Pull request merged**